console/src/components/UIKit/Icons/Icon.style.js

```diff
@@ -0,0 +1,10 @@
+import styled from 'styled-components';
+import { color, typography, layout, space } from 'styled-system';
+
+export const IconStyles = styled.span`
```

Comment:
IconStyles -> StyledIcon (or something like that)

Reply:
Noted.